inc/actions/namespace.php

```diff
@@ -202,3 +202,21 @@ function output_custom_colors() {
 function remove_admin_bar_callback() {
 	remove_action( 'wp_head', '_admin_bar_bump_cb' );
 }
+
+/**
+ * Hide content editor for Catalog page.
+ */
+function hide_catalog_content_editor() {
+	$post_id = isset( $_GET['post'] ) ? $_GET['post'] : null ;
```

Comment:
PHP7 supports this syntax:

`$post_id =  $_GET['post'] ?? null`

No need to change it, but in case you want a shorter version for the future.